### PR TITLE
Webapp

### PR DESCRIPTION
**Pull request by StarMindz**
Created a Single page App using the Book list class and concept

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -81,3 +81,39 @@ document.addEventListener('DOMContentLoaded', () => {
...
+  contact.className = 'contact display';
+  addForm.className = 'display-form nodisplay';
+  texttest.innerText = 'Contact Information';
+});
+addBtn.addEventListener('click', () => {
+  list.className = 'nodisplay';
+  contact.className = 'contact nodisplay';
+  addForm.className = 'display-form display';
+  texttest.innerText = 'Add a new book';
+});
```
Comment (Ismailco) [inline index.js:119]: - [ ] I can't navigate the site, kindly fix this so your app can have 3 sections that are appropriately displayed when a user clicks on the nav links

Screenshot_20220805_070945

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @StarMindz, @dgonzalesi, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by StarMindz** [comment]
@Ismailco Thank you for your review. The navigation links are all clickable. However, I've made changes so the add book is displayed by default and then you can navigate to the other sections by clicking the respective navigation links. 
Thank you very much

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle:
success

## Hey @StarMindz, @dgonzalesi  :wave:

Great work on implementing the changes requested by the previous reviewer. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:
:heavy_check_mark: GitHub Flow
:heavy_check_mark: Following the requirements

### [Optional]
## 1. 
Screenshot from 2022-08-05 11-57-32

- [ ] Your PR has a title that is not so descriptive enough. Because **webapp** is not describing what are you doing in the project. PR title fully describes what is happening in this branch. You could use **Awesome books: a complete website with navigation* for your PR title. :+1:

## 2
### Your Summary
summary-invalid


### Descriptive Summary Example :heavy_check_mark:
summary-example



- [ ] You don't have a descriptive summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking:

## 3.
- [ ] Your branch name is not descriptive enough. You are using **webapp**, you can use  **feature/awesome-books-navbar** :+1:

## 4.
### Your Linters
missing-linters
### Complete Linters
linter-complete

- [ ] You are missing webhint linter from your linters. Please add it because linters prevent your app from breaking. :blush:

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**